py3status/modules/spaceapi.py

```diff
@@ -57,10 +52,7 @@ def check(self):
                 self.closed_color = ''
 
             # grab json file
-            json_file = urllib.request.urlopen(self.url)
-            reader = codecs.getreader("utf-8")
-            data = json.load(reader(json_file))
-            json_file.close()
+            data = self.py3.get_json_data(self.url)
```

Comment:
Sorry, I should have paid a bit more attention to the whole function.

Yeah I hate the whole catch any exception stuff as it tends to hide problems more than it helps.  But as you say cleaning that up is probably outside of this PR.